src/commands/readme.js

```diff
@@ -16,34 +17,53 @@ module.exports.description = 'inject documentation into your README.md';
  * @returns {Object} yargs with options
  * @private
  */
-module.exports.builder = {
-  usage:
-    'Usage: documentation readme [--readme-file=README.md] --section "API"' +
-    ' [--compare-only] [other documentationjs options]',
-  example: 'documentation readme index.js -s "API Docs" --github',
-  'readme-file': {
-    describe: 'The markdown file into which to inject documentation',
-    default: 'README.md'
-  },
-  section: {
-    alias: 's',
-    describe:
-      'The section heading after which to inject generated documentation',
-    required: true
-  },
-  'diff-only': {
-    alias: 'd',
-    describe:
-      'Instead of updating the given README with the generated documentation,' +
-      ' just check if its contents match, exiting nonzero if not.',
-    default: false
-  },
-  quiet: {
-    alias: 'q',
-    describe: 'Quiet mode: do not print messages or README diff to stdout.',
-    default: false
+module.exports.builder = _.assign(
```

Comment:
Why use lodash here?

Reply:
I think it's to be consistent with how the same thing is done in `serve.js`, right next to `readme.js`: https://github.com/documentationjs/documentation/blob/11d9045a00a7565699b4f84148b99b56e810e84e/src/commands/serve.js#L21-L32